admin/installer/cloudformation.py

```diff
@@ -0,0 +1,293 @@
+"""
+Troposphere script to generate an AWS CloudFormation JSON template.
+
+Sample usage:
+python cloudformation.py > flocker-cluster.cloudformation.json
+
+Resulting JSON template has the following blueprint to describe the
+desired stack's resources and properties:
+1 Control Node with Flocker Control Service, (TLS-enabled) Swarm Manager,
+                    (TLS-enabled) Docker, Ubuntu 14.04
+2 Agent Nodes with Flocker Dataset Agent, Swarm Agent, (TLS-enabled) Docker,
+                   Ubuntu 14.04
+1 Client Node with Flockerctl, Docker, Docker-compose, Ubuntu 14.04
+
+To manifest the blueprint, please input the JSON template at AWS CloudFormation
+Create Stack console (after replacing ``us-east-1`` with your Region):
+https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new
+
+"""
```

Comment:
Thanks for the suggestion! Drafted https://github.com/ClusterHQ/flocker/blob/flocker-cloudformation-FLOC-3709/admin/installer/README.rst . Will get it reviewed by @wallrj .